### DISCUSSION TITLE
compiling error?

### DISCUSSION THREAD

**Discussion by daxue2017** [LizardByte/community#357] [Sunshine] [answered]
when to compile the sunshine by cmake, there is some errors below:

///////////////
ERROR: Got git error while fetching tags: 128
CMake Error at cmake/dependencies/common.cmake:7 (add_subdirectory):
  add_subdirectory given source "third-party/moonlight-common-c/enet" which
  is not an existing directory.
Call Stack (most recent call first):
  CMakeLists.txt:46 (include)
///////////

does i need to copy the moonlight-common-c/enet to the third-party?

**Comment by ReenigneArcher** [comment, marked as answer]
You need to clone sunshine with submodules recursively. https://docs.lizardbyte.dev/projects/sunshine/en/latest/building/build.html
